examples/complete-ecs/main.tf

```diff
@@ -0,0 +1,58 @@
+provider "aws" {
+  region  = "eu-west-1"
+  version = "v1.18.0"
+}
+
+provider "terraform" {}
+
+locals {
+  name = "my-ecs"
```

Comment:
I usually pass values similar to the name of the example to be able to have several examples at once - `complete-ecs` in this case.